tests/PKSim.Tests/IntegrationTests/IndividualToIndividualBuildingBlockMapperSpecs.cs

```diff
@@ -50,9 +52,33 @@ public void the_properties_of_the_building_block_should_match()
       public void should_have_replaced_the_ROOT_key_element_path_in_all_formula()
       {
          //this formula is defined as it is human specific
-         var formula = _buildingBlock.FormulaCache.FindByName("PARAM_eGFR");
+         var formula = _individualBuildingBlock.FormulaCache.FindByName("PARAM_eGFR");
          formula.ObjectPaths.Count.ShouldBeEqualTo(2);
          formula.ObjectPaths[0][0].ShouldBeEqualTo(Constants.ORGANISM);
       }
    }
+
+   public class When_mapping_an_individual_with_expression_to_individual : concern_for_IndividualToIndividualBuildingBlockMapper
+   {
+      public override void GlobalContext()
+      {
+         base.GlobalContext();
+
+         DomainFactoryForSpecs.CreateExpressionProfileAndAddToIndividual<IndividualEnzyme>(_individual, "CYP");
+      }
+
+      [Observation]
+      public void should_not_export_the_molecule_specific_ontogeny_factors()
+      {
+         var allOntogenyFactors = _individualBuildingBlock.Where(x => x.Name.IsOneOf(OntogenyFactors.ToArray()));
+         allOntogenyFactors.Count().ShouldBeEqualTo(0);
+      }
+
+      [Observation]
+      public void should_have_exported_the_plasma_protein_molecules()
```

Comment:
this test is not really required but I wanted to document that we are NOT removing the plasma protein stuff with this check